src/ShaderGen.Build/ShaderGen.targets

```diff
@@ -23,6 +23,7 @@
       <_SGArgs Condition="'$(ListAllShaderPaths)' == 'true'">$(_SGArgs) --listall</_SGArgs>
       <_SGArgs Condition="'$(ShaderProcessorAssembly)' != ''">$(_SGArgs) --processor "$(ShaderProcessorAssembly)"</_SGArgs>
       <_SGArgs Condition="'$(ShaderProcessorArgs)' != ''">$(_SGArgs) --processorargs "$(ShaderProcessorArgs)"</_SGArgs>
+      <_SGArgs Condition="'$(DebugShader)' == 'true' || '$(Configuration)' == 'Debug'">$(_SGArgs) --debug "$(ShaderProcessorArgs)"</_SGArgs>
```

Comment:
Sorry I didn't notice this before. `"$(ShaderProcessorArgs)"` isn't needed here -- I think it's just a copy-paste error from the previous line.

Reply:
Updated it